### ISSUE TITLE
Compatibility with other dimensions

### ISSUE THREAD

**Issue by Traincrafter111** [Johni0702/BetterPortals#325]
Hi I want to know if it would be possible to add support to the next mods:
Atum 2
Dimensional Doors
Erebus
TheBetweenlands
Mystcraft
The First 4 because I would love to see their portals getting better and mystcraft because it crash evn if it does not have portals it crash when its installed with BP well thats my request but if its not possible dont worry I know theres a lot of resources and time in mods and videogames :)

**Comment by Johni0702** [comment]
Pretty sure all of those except Mystcraft already have an issue. Please use the search functionality before opening a new one. IIRC DimDoors was blocked on BP 0.3 features (working and low-overhead same-world teleports specifically) and is now on the potentially-add-support-list (there was even interest from their side), all the others have licenses which do not grant me sufficient freedom to a) be allowed to add support or b) want to spend my time on them.

For the Mystcraft crash, please open a separate issue and provide the crash as well as any other details which might aid in reproducing it.
